src/lib/steps.ts

```diff
@@ -93,6 +93,24 @@ export type DeleteStep = {
   columns: string[];
 };
 
+export type StatisticsStep = {
+  name: 'statistics';
+  column: string;
+  statistics: (
```

Comment:
After trying the step several times, I strongly believe it's a pain to have all statistics checked by default. It's quite overwhelming and in most cases I just want to have a look at 2 or 3 of them max, so I would prefer not to have anything checked by default and pick what I need